ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -252,6 +253,7 @@ def get_workflow(dict_configs, cdump='gdas'):
     do_wafs = base.get('WAFSF', 'NO').upper()
     do_vrfy = base.get('DO_VRFY', 'YES').upper()
     do_metp = base.get('DO_METP', 'NO').upper()
+    n_tiles = 6
```

Comment:
This is the # of FV3 tiles, right? If so, this seems like something we should add into a config file and then source from there (?).

Reply:
Yes and yes, though I was fine hard-coding it since I don't expect it to change unless/until we have to worry about nests.

Reply:
Best location would be config.base, since it pops up a bunch of places. Right now it is hard-coded in all of them.

Reply:
Replacing all the places we have 6 with a centralized variable for number of tiles needs its own PR, if we are going to bother.